src/js/modules/rpc/errors.ts

```diff
@@ -115,7 +115,10 @@ const createResponseScriptWithoutTopics: HandleEnableResponse = (handle) => ({
 });
 
 const createResponseScriptSyntaxError: SimpleEnableResponse = (handleDef) => ({
-  enableResponseCode: EnableResponseCodes.scriptContainsSyntaxError,
+  // the enableResponseCode should be:
```

Comment:
You can put the - fixes: #pr in the commit message so its always in the source tree. Its also automatically detected by github